src/maptm/src/maptm_config.c

```diff
@@ -450,8 +481,14 @@ bool config_mapt(void)
         return false;
     }
 
-    /* Hard Coded value: Must be changed */
-    char subnetcidr4[20] = "192.168.1.1/24";
+    // Get LAN Subnet
+    char subnetcidr4[20] = "";
+    set_subnetcidr4(subnetcidr4);
+    if (subnetcidr4 == NULL)
```

Comment:
`subnetcidr4` cannot be NULL, it's on stack, so check the return value:

```suggestion
    if (get_subnetcidr4(subnetcidr4) == NULL)
```

Also changed `set_subnetcidr4` to `get_subnetcidr4` (in line with the suggestion above).